internal/sudo/contract_manager.go

```diff
@@ -75,37 +74,72 @@ type OpenAckDetails struct {
 	CounterpartyVersion   string `json:"counterparty_version"`
 }
 
+type ContractMethods interface {
```

Comment:
do we really need this interface? seems like it can be removed and wasmKeeper used as it is inside the ContractManager

Reply:
This approach will simplify testing and to add additional limits to wasmKeeper methods , otherwise it will be required to use something like mockery to prepare objects for testing inside unit test. But maybe there is more convenient way to pass keeper object and to mock/test this keeper's methods.

Reply:
> I personally don't like the name. it looks like an enum, but is an interface

Let's discuss it